doc/source/index.rst

```diff
@@ -59,6 +59,8 @@ Questions and discussions about MDAnalysis take place on `GitHub Discussions`_ a
 
 Wherever possible, do not take these conversations to private channels, including contacting the maintainers directly. Keeping communication public means everybody can benefit and learn from the conversation.
 
+.. _`mdnalysis-discussion`:
+   http://groups.google.com/group/mdnalysis-discussion
```

Comment:
This is no longer used. Is it referenced anywhere in the page? If the reference is no longer relevant, it should be removed.

Reply:
Then instead of google group, shall I add about [discord](https://discord.gg/fXTSfDJyxE) channel [here](https://github.com/MDAnalysis/UserGuide/blob/b0da13f66a5ca13ae0c0e4372c77b30628695325/doc/source/index.rst?plain=1#L42C138-L42C158)?

Reply:
Yes, that would be great. I would add GitHub Discussion though, not the Discord server. though, it is more appropriate to ask specific questions on GitHub so we have them all in the same place.